v0.1/core/annotation.schema.json

```diff
@@ -23,21 +23,28 @@
       "description": "Metadata node identifier (id).",
       "pattern": "sands/v0.1/core/annotation/([A-Za-z0-9_-]+).json"
     },
+    "alias": {
+      "type": "string",
+      "description": "User-defined identifier, often a short name, of this annotation."
+    },
     "criteria": {
       "oneOf": [{
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/PROTOCOL"
       }, {
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/PROTOCOL_EXECUTION"
       }],    
-      "description": "criteria"
+      "description": "Criteria used to produce the annotation."
     },
     "criteriaType": {
       "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
       "description": "How was this annotation determined."
     },
-    "definingTerm": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANATOMICAL_ENTITY",
-      "description": "Anatomical entity that provides the term for the here specified annotation."
+    "namingTerm": {
+      "type": "array",
+      "description": "Anatomical entity (term) that provides the term for the here specified annotation.",
```

Comment:
suggestion for description:
`"Anatomical entity (anatomical term) that names the here specified annotation."`